### PR TITLE
php 7.4 and fix   str_repeat(): Second argument has to be greater than or equal to 0

### PR DESCRIPTION
**Pull request by elsayed85**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-    "name": "wulfheart/pretty_routes",
+    "name": "elsayed85/pretty_routes",
```
Comment (Gummibeer) [inline composer.json]: The vendor of that package doesn't change.

File: composer.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     "description": "This is my package PrettyRoutes",
     "keywords": [
         "Wulfheart",
         "laravel",
         "pretty_routes"
     ],
-    "homepage": "https://github.com/wulfheart/pretty_routes",
+    "homepage": "https://github.com/elsayed85/pretty_routes",
```
Comment (Gummibeer) [inline composer.json]: Homepage keeps the same

**Review by Gummibeer — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (elsayed85) [reply composer.json]: done

File: composer.json (hunk 2 above)
Comment (elsayed85) [reply composer.json]: done

**Comment by Wulfheart** [comment]
Thank you for your PR. However, the changes got merged via other PRs.

**Pull request closed without merging**